_pages/releases/process/index.md

```diff
@@ -3,31 +3,104 @@ layout: content-page
 title: Release Process
 ---
 
-> The release process can only be executed by core contributors with access to dependency systems.
```

Comment:
It would be good to keep this available somewhere in case the automated system breaks, etc.... maybe as a sub-page with a link at the bottom of this?